pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -80,6 +80,9 @@ type Reconciler interface {
 // nodeName - the Name for this node, used by Attach and Detach methods
 // desiredStateOfWorld - cache containing the desired state of the world
 // actualStateOfWorld - cache containing the actual state of the world
+// populatorHasAddedPods - checker for whether the populator has finished
+//   adding pods to the desiredStateOfWorld cache at least once after sources
```

Comment:
I'm copying the other multi-line parameter comments which are indented by 2 spaces if you mean the spaces at the beginning